### PR TITLE
[DE] Update `Gameplay/Judgement/Slider break` 

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Gameplay/Judgement/Slider_break/de.md [hunk 1]
```diff
@@ -7,12 +7,14 @@ tags:
...
 ---
 
 # Sliderbreak
 
-**Sliderbreak** (manchmal abgekürzt als *SB*) ist ein Begriff, der verwendet wird, um das Unterbrechen einer Combo durch einen teilweise vollendeten [Slider](/wiki/Gameplay/Hit_object/Slider) zu beschreiben. Sliderbreaks können auf drei verschiedene Arten auftreten: den Kopf eines Sliders zu verfehlen, während der Körper noch ganz oder teilweise fertiggestellt ist, einen Slider zu irgendeinem Zeitpunkt seiner Dauer loszulassen oder nicht alle Ticks des Sliders zu sammeln. Alle drei Arten werden zusammen als "Sliderbreak" bezeichnet und beenden die Combo des Spielers. Ein Sliderbreak während eines Durchlaufs führt zu einem "100er" und unterbricht die Combo, zählt aber nicht als Miss.
+**Sliderbreak** (manchmal abgekürzt als *SB*) ist ein Begriff, der verwendet wird, um das Unterbrechen einer [Combo](/wiki/Gameplay/Combo_(score_multiplier)) durch einen unvollendeten [Slider](/wiki/Gameplay/Hit_object/Slider) zu beschreiben. Die Unterbrechung tritt auf, wenn entweder der [Anfang](/wiki/Gameplay/Hit_object/Slider/Sliderhead) oder eines der [Ticks](/wiki/Gameplay/Hit_object/Slider/Slider_tick) des Sliders verfehlt werden.
```
Comment (respektive) [inline wiki/Gameplay/Judgement/Slider_break/de.md:14]:
```suggestion
**Sliderbreak** (manchmal abgekürzt als *SB*) ist ein Begriff, der verwendet wird, um das Unterbrechen einer [Combo](/wiki/Gameplay/Combo_(score_multiplier)) durch einen unvollendeten [Slider](/wiki/Gameplay/Hit_object/Slider) zu beschreiben. Die Unterbrechung tritt auf, wenn entweder der [Anfang](/wiki/Gameplay/Hit_object/Slider/Sliderhead) oder einer der [Ticks](/wiki/Gameplay/Hit_object/Slider/Slider_tick) des Sliders verfehlt werden.
```
I think tick should be masculine? dunno kinda hard to tell in german.
Alternatively could also just say `oder ein Tick des Sliders verfehlt wird` maybe.

**Review by respektive — CHANGES_REQUESTED** [review]

File: wiki/Gameplay/Judgement/Slider_break/de.md (hunk 1 above)
Comment (The-Last-Cookie) [reply wiki/Gameplay/Judgement/Slider_break/de.md:14]: If I don't have a bend in my optics, Genitiv Maskulin of `ein` is indeed `eines`. Everything else looks wrong to me.

See [verbformen.de](https://www.verbformen.de/deklination/artikel/ein.htm) and [korrekturen.de](https://www.korrekturen.de/flexion/adjektive/ein/).

Comment (respektive) [reply wiki/Gameplay/Judgement/Slider_break/de.md:14]: Oh that's weird. I must have always done that wrong then. This just looks so wrong to me, but I guess it is correct then. man gendered languages suck, even as a native I get it wrong :skull:

**Review by respektive — APPROVED** [review]
Seems that I'm just stupid and this is actually correct.

**Review by TicClick — APPROVED** [review]

**Pull request merged**